README.md

```diff
@@ -74,14 +74,17 @@ Clone this repository by using the command line :
 
 Install this project with:
 
-`npm gemfile`
+`bundle install `
 
 ### Usage
 
 To run the project, execute the following command:
 
 `rails server`
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you improve your `## Getting Stared` section by referring this **[example](https://gist.github.com/tresorsawasawa/4cde69cff1b40161b9de380f9891f415)** to make your documentation not only more `descriptive` and `professional` but also to help non technical users to have all information about using your project locally.